tests/Unit/ProtoTests/ProtoTest.php

```diff
@@ -122,7 +122,7 @@ public function testDeprecatedService(): void
 
     public function testBasicDiregapic(): void
     {
-        $this->runProtoTest('BasicDiregapic/library_rest.proto', 'google.example.library.v1', 'rest');
+        $this->runProtoTest('BasicDiregapic/library_rest.proto', 'google.example.library.v1', 'rest', migrationMode: MigrationMode::PRE_MIGRATION_SURFACE_ONLY);
```

Comment:
This is for sure nits, but maybe we should rename these tests based on the migration mode, just `testMigrationModePreMigrationSurfaeOnly()` and so on and so forth?

Reply:
Sure thing. I named them `test{OriginalName}WithPreMigrationSurface` and so on, because ultimately we don't want to lose that the test was originally for, and because we will remove `migration_mode` in generator v2 (once v1 is dropped).

Reply:
thanks! that makes sense